internal/db/queries.sql

```diff
@@ -101,3 +101,9 @@ RETURNING repo_sync_queue_id
 DELETE FROM public.repos WHERE repo_import_id = $1::uuid AND NOT(repo = ANY($2::TEXT[]))
 ;
 
+-- name: CleanLogs :exec
+SELECT mergestat.simple_log_retention($1::INTEGER);
```

Comment:
```suggestion
SELECT mergestat.simple_repo_sync_log_retention($1::INTEGER);
```

Just a suggestion to be overly verbose with our naming here to be extra clear what this function/statement does. Might make it a bit easier in a couple months if we're looking at this again to recognize what's happening here more quickly